app/controllers/health_check.go

```diff
@@ -1,7 +1,9 @@
 package controllers
 
-import "net/http"
+import (
```

Comment:
Single imports should be one liners. The standard goimports should do this for you.